src/main/java/io/vlingo/http/resource/RequestHandler0.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL
+ * was not distributed with this file, You can obtain
+ * one at https://mozilla.org/MPL/2.0/.
+ */
+
+package io.vlingo.http.resource;
+
+import io.vlingo.http.Header;
+import io.vlingo.http.Method;
+import io.vlingo.http.Request;
+import io.vlingo.http.Response;
+
+import java.util.Collections;
+
+public class RequestHandler0 extends RequestHandler {
```

Comment:
Is there no other way than `0-5` or at least reduce the repeated code? What about varargs?

`See:`
https://github.com/vlingo/vlingo-actors/blob/master/src/main/java/io/vlingo/actors/Protocols.java